### PR TITLE
Moving Cloud Security content from InSpec repo

### PR DESCRIPTION
**Pull request by dkumaras**
Signed-off-by: Deepa Kumaraswamy <dkumaras@progress.com>

## Description

[Please describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/cloud_security.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+- Shared controls - Responsibilities that vary depending on the service model.
+
+  - Infrastructure-as-a-Service (IaaS)): A hybrid approach where organizations can manage some of their data and applications on-premise while relying on cloud providers to manage servers, hardware, networking, virtualization, and storage needs.
+
+  - Platform-as-a-Service (PaaS): Provides organizations the ability to streamline their application development and delivery by providing a custom application framework that automatically manages operating systems, software updates, storage, and supporting infrastructure in the cloud.
+
+  - Software-as-a-Service (SaaS): Cloud-based software hosted online and typically available on a subscription basis. Third-party providers manage all potential technical issues, such as data, middleware, servers, and storage, minimizing IT resource expenditures and streamlining maintenance and support functions.
```
Comment (IanMadd) [inline content/cloud_security.md]: These shouldn't be hyphenated. See: https://developers.google.com/style/word-list#letter-i

Comment (dkumaras) [reply content/cloud_security.md]: Ok - I have removed them. Actually, MSTP and CMS say to do.

Comment (dkumaras) [reply content/cloud_security.md]: Would you please merge it?

**Comment by IanMadd** [comment]
This needs additional review.

**Comment by IanMadd** [comment]
Closing this because this isn't the right content for documentation.

**Pull request closed without merging**